app.json

```diff
@@ -39,6 +39,14 @@
       "description": "Algolia Places API key",
       "required": false
     },
+    "ATHENA_WORK_GROUP": {
+      "description": "work group for Athena queries",
+      "required": false
+    },
+    "ATHENA_REGION_NAME": {
+      "description": "Region name for Athena queries",
+      "required": false
+    },
```

Comment:
As part of setting up the permissions for Open to access Athena I have set up per-environment workgroups and database names, so there should be an additional parameter of `"ATHENA_DATABASE_NAME"` that is used to determine which database to execute queries against.